frontend/components/transaction/BidTransactionTable.vue

```diff
@@ -168,6 +168,17 @@ export default Vue.extend({
         isBidAmountNaN(): boolean {
             return !!this.amountToBid?.isNaN();
         },
+        isTooSmallToPartiallyTake(): boolean {
+            return this.auctionTransaction?.totalPrice.isGreaterThanOrEqualTo(this.minimumDepositDai?.multipliedBy(2));
+        },
```

Comment:
Why do you need to check it again here? I thought in this case, BidInput should emit NaN?

Reply:
I wanted to disable the `Buy all collateral` and `Set minimum bid` buttons in this case, but using the `NaN` check here would also disable it when errors occur so this was the only way I figured out to isolate it.